doc/src/guides/solving/solving-guidance.md

```diff
@@ -0,0 +1,221 @@
+# Solving Guidance
+
+These guidelines apply to many types of solving.
+
+## Numeric Solutions
+
+### Numeric Solutions by Choice
+Solving functions such as {func}`~.solve` and {func}`~.solveset` will not try to
+find a numeric solution, only a mathematically-exact symbolic solution. So if
+you want a numeric solution, consider {func}`~.nsolve`.
+
+In some situations, even though a closed-form solution is available, it may be
+too cumbersome to be desirable. In that case, you can use {func}`evalf n
+<sympy.core.evalf>` instead if a numerical solution is acceptable:
+
+```py
+>>> from sympy import symbols, solve
+>>> x = symbols('x')
+>>> solutions = solve(x**4 + 10*x**2 + x + 1, x, dict=True); solutions
```

Comment:
I'm not a fan of the semicolon thing in doctest examples. Just write `>>> solutions` on another line.